packages/flutter_tools/templates/app/linux.tmpl/CMakeLists.txt.tmpl

```diff
@@ -79,6 +79,11 @@ if(PLUGIN_BUNDLED_LIBRARIES)
     COMPONENT Runtime)
 endif()
 
+install(FILES app.desktop DESTINATION "${INSTALL_BUNDLE_DATA_DIR}/applications" RENAME "{{projectName}}.desktop"
```

Comment:
It might make sense to use `{{linuxIdentifier}}` once https://github.com/flutter/flutter/pull/60053 is landed.